x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/plugin/BasicFormatter.java

```diff
@@ -4,11 +4,12 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-package org.elasticsearch.xpack.sql.action;
+package org.elasticsearch.xpack.sql.plugin;
```

Comment:
Why?

Reply:
I had to move it out of the `sql-action` project to make it extend `FormatterState`. While this won't be necessary anymore, I think it still makes sense to keep it out of `sql-action`. I guess at some point there was another project using it but that doesn't seem to be the case anymore. The CLI depends directly on `SimpleFormatter` so that also can't be the reason.